docs/core/diagnostics/compare-metric-apis.md

```diff
@@ -44,7 +29,7 @@ Although this API was designed to work well with OpenTelemetry and its growing e
 
 ### PerformanceCounter
 
-<xref:System.Diagnostics.PerformanceCounter?displayProperty=nameWithType> APIs are the oldest metric APIs. They're only supported on Windows and provide
+<xref:System.Diagnostics.PerformanceCounter?displayProperty=nameWithType> APIs are the oldest .NET metric APIs. They're only supported on Windows and provide
```

Comment:
> .NET [](http://example.com/codeflow?start=93&length=5)

this will be excluding the .NET Framework?

Reply:
It wasn't intended to exclude .NET Framework. I could remove it if you think it creates confusion?